index.html

```diff
@@ -26,33 +26,42 @@
       <a class="py-2 px-4" onclick="displayMenu()" href="#"><h2>CTF Campaigns</h2></a>
     </div>
   </div>
-
+  
+  <button id="menu-button" class="hamburger mobile-only absolute" type="button">
+      <div></div><div></div><div></div>
```

Comment:
Kindly removed unused tags to clean up the code.

Reply:
this tags are used to create the hamburger button